src/main/scripts/org/reaktivity/specification/sse/challenge/respond.to.challenge/response.rpt

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property serverTransport "nukleus://streams/target#0"
+
+accept ${serverTransport}
+       option nukleus:capabilities ${core:capabilities("CHALLENGE")}
+       option nukleus:window 8192
+       option nukleus:transmission "half-duplex"
+accepted
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":scheme", "http")
+                             .header(":method", "GET")
+                             .header(":path", "/events/a8b7c6d5")
+                             .header(":authority", "localhost:8080")
+                             .header("accept", "text/event-stream")
+                             .build()}
+
+connected
+
+read nukleus:challenge ${http:challengeEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":method", "POST")
+                             .header("content-type", "application/x-challenge-response")
+                             .build()}
+write "type:challenge\n\ndata:{\"method\": \"POST\", \"headers\": {\"content-type\": \"application/x-challenge-response\"}}\n\n"
```

Comment:
Just one `\n` after `type:challenge` since `\n\n` marks end of SSE event but both `type` and `data` are part of the same SSE event.
```
type:challenge\n
data:{\"method\": \"POST\", \"headers\": {\"content-type\": \"application/x-challenge-response\"}}\n
\n
```